run-tests.sh

```diff
@@ -20,3 +20,7 @@ set -ev
 ./gradlew -p examples/test1 --refresh-dependencies --rerun-tasks clean check
 
 ./gradlew -p examples/test2 --refresh-dependencies --rerun-tasks clean check
+
+./gradlew -p examples/test3 --refresh-dependencies --rerun-tasks clean compileThrift
```

Comment:
Minor) Would you mind giving a proper name to the test folder to easily no the intention of the tests?